### PR TITLE
feat: Show correct currency in Tickets Fragment

### LINKED ISSUE #311: Show correct currency in Tickets Fragment
Right now it is always $

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #311 
screenshot_20180712-213120

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            try {
+                val currency = Currency.getInstance(locale)
+                currencyLocaleMap.put(currency, locale)
+            } catch (e: NullPointerException) {
+                Timber.d(e, "Failed!")
+            }
+        }
+    }
+
+    fun getCurrencySymbol(currencyCode: String?): String? {
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt]: Same method should be used everywhere to convert into the symbol. 
Is this more accurate than the one used before? If yes use this method in the attendee fragment as well.

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt]: Sure :+1:

File: app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                currencyLocaleMap.put(currency, locale)
+            } catch (e: IllegalArgumentException) {
+                Timber.d(e, "Failed!")
+            }
+        }
+    }
+
+    fun getCurrencySymbol(currencyCode: String?): String? {
+        val currency = Currency.getInstance(currencyCode)
+        return currency.getSymbol(currencyLocaleMap[currency])
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt]: I think there is no need of creating a util class for this
 `currency.symbol` does the same thing what this entire util class is doing.

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt]: Okay :+1:

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt]: @nikit19 Updated PR

File: app/src/main/java/org/fossasia/openevent/general/ticket/TicketViewHolder.kt [hunk 3]
```diff
@@ -30,9 +30,15 @@ class TicketViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
             itemView.orderRange.adapter = ArrayAdapter(itemView.context, R.layout.select_dialog_singlechoice, spinnerList)
         }
 
+        val price = StringBuilder()
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/ticket/TicketViewHolder.kt]: why not store the currency in a different variable and reference it using $ in `  itemView.price.text`
using the stringBuilder makes it look a little complicated

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/ticket/TicketViewHolder.kt]: Updated

**Comment by iamareebjamal** [comment]
Why pass it around through bundles when it can be calculated?

**Comment by dreadpool2** [comment]
@iamareebjamal Actually the currency used is present in the event object, not present in the ticket object. So I am passing the currency present in the Event object by a bundle to the Tickets fragment

**Comment by iamareebjamal** [comment]
That's not good

**Comment by dreadpool2** [comment]
@iamareebjamal Could you think of any ideal solution to this? I am not able to find any other alternate :(

**Comment by iamareebjamal** [comment]
OK, right now we'll use this. Create an issue to handle this later

**Comment by dreadpool2** [comment]
Okay will do that :+1:

**Comment by dreadpool2** [comment]
@iamareebjamal Created an issue :+1:

**Comment by nikit19** [comment]
@dreadpool2 Looks good to me! could you also convert the currency in ticket details too. Otherwise this is ready to be merged

**Comment by dreadpool2** [comment]
Doing that :+1:

**Pull request merged**